src/main/resources/static_config/static_roles.yml

```diff
@@ -33,6 +33,9 @@ kibana_user:
         - ".kibana"
         - ".kibana-6"
         - ".kibana_*"
+        - ".opensearch_dashboards"
+        - ".opensearch_dashboards-6"
```

Comment:
I have no context, but could you help me understand where we would use `.opensearch_dashboards-6`?

Reply:
Neither do I have the idea. It could be some legacy in kibana and I suggest it will be safer to keep it

Reply:
`.kibana-6` came out of [migration steps to 6.x](https://www.elastic.co/guide/en/kibana/6.0/migrating-6.0-index.html) provided by Kibana (6.7 in case of security plugin). 

Is similar plan supported by OpenSearch ?

Reply:
@sujithvm Are you referring to the plan to keep backward compatibility in OpenSearch?
My understanding is that the plan hasn't been dicussed yet, so not clear yet. @saratvemulapalli Could you please confirm this?

Reply:
The migration plan is TBD. We dont know how it is going to look like and whats needed. If you have any thoughts or suggestions please feel free to add them to the issues.

Ref: 
https://github.com/opensearch-project/OpenSearch/issues/640
https://github.com/opensearch-project/OpenSearch/issues/638

Reply:
As @saratvemulapalli we're still chewing on backwards compatibility.  I agree with @hsiang9431-amzn that we should leave dashboard-6 in place and tackle it when we  make other backwards compatibility changes.